packages/concerto-cto/package.json

```
@@ -56,7 +56,7 @@
     "typescript": "4.6.3"
   },
   "dependencies": {
-    "@accordproject/concerto-metamodel": "3.9.1",
+    "@accordproject/concerto-metamodel": "3.10",
```

I have another PR to update this. So don't worry about it. But the change should be 
```suggestion
    "@accordproject/concerto-metamodel": "3.10.0",
```